pkg/asset/cluster/cluster.go

```diff
@@ -3,12 +3,14 @@ package cluster
 import (
 	"context"
 	"fmt"
+	"os"
 	"path/filepath"
 	"strings"
 
 	"github.com/pkg/errors"
 	"github.com/sirupsen/logrus"
 
+	"github.com/openshift/installer/cmd/openshift-install/command"
```

Comment:
I am going to test adding the install dir to the context, to reduce our dependence on this package/global variable.

Reply:
nevermind. there is also a variable to avoid this. will capture this in jira for followup